boards/esp32-ethernet-kit-v1_0/include/periph_conf.h

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ * Copyright (C) 2020 Google LLC
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_esp32_esp-ethernet-kit
+ * @brief       Peripheral MCU configuration for Espressif ESP32-Ethernet-Kit
+ * @file
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ * @author      Erik Ekman <eekman@google.com>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+ extern "C" {
+#endif
+
+/**
+ * @name    ADC and DAC channel configuration
+ * @{
+ */
+
+/**
+ * @brief   Declaration of GPIOs that can be used as ADC channels
+ *
+ * @note As long as the GPIOs listed in ADC_GPIOS are not initialized as ADC
+ * channels with the `adc_init` function, they can be used for other
+ * purposes.
+ */
+#ifndef ADC_GPIOS
+#define ADC_GPIOS   { GPIO34, GPIO35, GPIO36, GPIO39 }
+#endif
+
+/**
+ * @brief   Declaration of GPIOs that can be used as DAC channels
+ *
+ * ESP32-Ethernet-Kit has no GPIOs left that might be used as DAC channels.
+ */
+#ifndef DAC_GPIOS
+#define DAC_GPIOS   { }
+#endif
+/** @} */
+
+/**
+ * @name   I2C configuration
+ *
+ * @note The GPIOs listed in the configuration are only initialized as I2C
+ * signals when module `periph_i2c` is used. Otherwise they are not
+ * allocated and can be used for other purposes.
+ *
+ * @{
+ */
+#ifndef I2C0_SPEED
+#define I2C0_SPEED  I2C_SPEED_FAST  /**< I2C bus speed of I2C_DEV(0) */
+#endif
+#ifndef I2C0_SCL
+#define I2C0_SCL    GPIO33          /**< SCL signal of I2C_DEV(0) */
+#endif
+#ifndef I2C0_SDA
+#define I2C0_SDA    GPIO32          /**< SDA signal of I2C_DEV(0) */
+#endif
+/** @} */
+
+/**
+ * @name   PWM channel configuration
+ *
+ * @note As long as the according PWM device is not initialized with
+ * the `pwm_init`, the GPIOs declared for this device can be used
+ * for other purposes.
+ *
+ * @{
+ */
+#ifndef PWM0_GPIOS
+#ifdef MODULE_ESP32_ETHERNET_KIT_V1_0
```

Comment:
Now that we have separate boards, we can use the macro `BOARD_*` directly. It makes the definition and use of the according pseudomodules obsolete.
```suggestion
#ifdef BOARD_ESP32_ETHERNET_KIT_V1_0
```